core/spec/models/spree/wallet/add_payment_sources_to_wallet_spec.rb

```diff
@@ -13,5 +14,28 @@
         order.user.wallet.wallet_payment_sources.count
       }.by(1)
     end
+
+    context "when the default wallet payment source is used and more a recent wallet payment exists" do
+      before do
+        credit_card_one = user.wallet.add(create(:credit_card, user: user))
+        user.wallet.add(create(:credit_card, user: user))
+        user.wallet.default_wallet_payment_source = credit_card_one # must be the first created card
+        order.payments.first.update!(source: credit_card_one.payment_source)
+      end
+
+      it 'does not make a new wallet payment source' do
+        expect(user.wallet).to receive(:add).and_call_original
+        expect { subject.add_to_wallet }.to_not change {
+          order.user.wallet.wallet_payment_sources.count
+        }
+      end
+
+      it 'does change the default wallet payment source' do
+        expect(user.wallet).to receive(:add).and_call_original
+        expect { subject.add_to_wallet }.to_not change {
+          user.wallet.default_wallet_payment_source
```

Comment:
The spec description says `does change the default wallet payment source` but seems like it's testing the opposite thing:  `...to_not change { user.wallet.default_wallet_payment_source }`. Am I missing something or something needs to be switched here?

Reply:
Whoops! 😅 Typo. Good catch, thanks!